.dangerfile/readme_tests.rb

```diff
@@ -277,31 +279,48 @@ def readme_invalid_credentials?(file, file_lines)
         if !line.start_with?("  - ")
           permission_list_found = 2
         else
-          asterix_found = 1 if line.strip.end_with?("*")
+          footnote_symbols["*"] = true if line.strip.end_with?("*")
+          footnote_symbols["†"] = true if line.strip.end_with?("\u2020")
+          footnote_symbols["‡"] = true if line.strip.end_with?("\u2021")
```

Comment:
Wondering instead of characters, if we decided to use a different footnote schema.  From wikis, they use
`[^digit]`.

```
This is a sample text with some footnotes [^1][^2][^3].

[1] Some text here
[2] More text
[3] Lorem ipsum
```

With that schema, we can use regex to check/match in the test more easily:
`/\[\^(\d+)\]/`
I think it'll scale out.. we have some PTs with 3 footnotes, but potentially we'll need another in future and how will we decide next character each time?

Reply:
I was using this - https://en.wikipedia.org/wiki/Note_(typography)#:~:text=Typographic%20characters%2C%20such%20as%20the,sign%20%C2%A7%2C%20(v)%20the

We can use numbers instead as agree it's probably easier to check (symbols look cooler though😉)

Reply:
This will turn into quite a large PR though, as we will need to update all READMEs and also Dangerfile readme script/policy permissions script

Reply:
> I was using this - https://en.wikipedia.org/wiki/Note_(typography)#:~:text=Typographic%20characters%2C%20such%20as%20the,sign%20%C2%A7%2C%20(v)%20the

This works too if we want to minimize scope for this change.. A peer-reviewed list of characters with at least 6 defined checks the box for what we need now imo ✅ 

> (i) the Asterisk [*]
> (ii) the Dagger mark †
> (iii) the Crossed dagger mark ‡
> (iv) the Section sign §
> (v) the Vertical bar ‖
> (vi) the Pilcrow ¶